apps/wasm/mvp/utils.spec.ts

```diff
@@ -0,0 +1,4 @@
+// Placeholder test
+test("1 + 1 = 2", () => {
+    expect(1 + 1).toBe(2);
```

Comment:
I think we can safely assume that this will continue to be true. Although with javascript you never really know :P